### ISSUE TITLE
Move to predefined node (by key command or toolbar button)?

### ISSUE THREAD

**Issue by metal450** [giuspen/cherrytree#2624]
Is there any way to assign either a hotkey or a toolbar button to "move the current node to a predefined node"?

Use-case: I have a node that I use as "Archived" (like "Trash" - I never actually delete things, I just move them there). Currently, whenever I want to remove a node, I have to do so manually: right click->Node Move->Node Change Parent->scroll through to manually find the destination node. It would be great if I could just assign a hotkey. Second best would be a toolbar button.

**Comment by gitvectors** [comment]
Workaround suggestion from this user (not developer):
File > Export > export to CherryTree > archive location (archive as individual .ctd docs)

then ...

Right Click on selected node in Tree Explorer > Delete Node

This gives archives as a list of *.ctd (dated) docs rather than a single *.ctd doc.
One approach is to then index them / query them using Recoll.
Or ripgrep-all.
Optionally you can consolidate them into one.  Archive.ctd.

**Comment by metal450** [comment]
The goal wasn't to move them into a separate .ctd doc - just to make the process of moving the node from one tree to another require fewer clicks. Entirely different end result.

**Comment by gitvectors** [comment]
Fair enough .. but I can do that easily if "Archived" node is topmost in tree explorer.  No need for scrolling. Other than that I can build an external AutoKey or similar Ui automation script. That is, outside CT.
